.cargo/config.toml

```diff
@@ -27,7 +36,15 @@ rustflags = [
   "--cfg", 'target_has_atomic="ptr"',
 ]
 
-target = "riscv32imc-unknown-none-elf"
+[target.xtensa-esp32s3-none-elf]
+runner = "espflash flash --baud 921600 --monitor --port /dev/ttyUSB1"
```

Comment:
looks good and I think we should merge it - just one nitpick:
hardcoding baud-rate and port isn't that good - better to use the env variables ESPFLASH_BAUD and ESPFLASH_PORT locally and leave them out in the runner

Reply:
Looks like this one sneaked in as I was doing my tests. Good catch.